### ISSUE TITLE
User input

### ISSUE THREAD

**Issue by dkedar7** [dkedar7/IDEAR-Dash#3]
Currently, target variables, categorical and numerical variables have to be manually set before analysis.

Make a new tab (home page) that will have the following functionality:
- Request the user to submit a CSV/ Excel file
- Allow them to set target, categorical and numerical variables
- 'Analyze' button triggers analysis and takes the user to the 'Profile Data' page.

@Vaishali-Guru , let me know if this makes sense. I see this as a good next step. I will take a shot at this in the next few weeks.

**Comment by Vaishali-Guru** [comment]
Yes, this would definitely be a good next step. It would improve the user interaction. 

@dkedar7 But, I do have a question: **Which option would be better to pass the target, categorical and numerical variable values? 
Passing directly to the function OR writing it to a file and then fetch it?**

**Comment by dkedar7** [comment]
@Vaishali-Guru We should try to pass these values directly using a callback function, rather than save them to a file locally.
